[2025-03-03T13:02:10.697+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: xkcd_dbt_transformations.create_wrapper manual__2025-03-03T12:50:23.237075+00:00 [queued]>
[2025-03-03T13:02:10.701+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: xkcd_dbt_transformations.create_wrapper manual__2025-03-03T12:50:23.237075+00:00 [queued]>
[2025-03-03T13:02:10.701+0000] {taskinstance.py:1359} INFO - Starting attempt 2 of 3
[2025-03-03T13:02:10.706+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): create_wrapper> on 2025-03-03 12:50:23.237075+00:00
[2025-03-03T13:02:10.708+0000] {standard_task_runner.py:57} INFO - Started process 536 to run task
[2025-03-03T13:02:10.710+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'xkcd_dbt_transformations', 'create_wrapper', 'manual__2025-03-03T12:50:23.237075+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/xkcd_dbt_transformations.py', '--cfg-path', '/tmp/tmpcv6f_vb8']
[2025-03-03T13:02:10.711+0000] {standard_task_runner.py:85} INFO - Job 104: Subtask create_wrapper
[2025-03-03T13:02:10.731+0000] {task_command.py:415} INFO - Running <TaskInstance: xkcd_dbt_transformations.create_wrapper manual__2025-03-03T12:50:23.237075+00:00 [running]> on host e42a692d6092
[2025-03-03T13:02:10.760+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='xkcd_dbt_transformations' AIRFLOW_CTX_TASK_ID='create_wrapper' AIRFLOW_CTX_EXECUTION_DATE='2025-03-03T12:50:23.237075+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-03T12:50:23.237075+00:00'
[2025-03-03T13:02:10.760+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-03T13:02:10.761+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\ncat > /tmp/dbt_wrapper.sh << \'EOF\'\n#!/bin/bash\n# Get the full path to dbt\nDBT_PATH="/home/***/.local/bin/dbt"\n\n# Run dbt with all arguments passed to this script\n$DBT_PATH "$@"\nexit_code=$?\n\n# Check if exit code is 1 (error)\nif [ $exit_code -eq 1 ]; then\n    # If we find "Error from git" in logs, treat as success\n    if dmesg | grep -q "Error from git" || grep -q "Error from git" /opt/***/logs/**/dbt_*.log 2>/dev/null; then\n        echo "Ignoring Git check error"\n        exit 0\n    fi\nfi\n\n# Return the original exit code\nexit $exit_code\nEOF\nchmod +x /tmp/dbt_wrapper.sh']
[2025-03-03T13:02:10.764+0000] {subprocess.py:86} INFO - Output:
[2025-03-03T13:02:10.767+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-03T13:02:10.777+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=xkcd_dbt_transformations, task_id=create_wrapper, execution_date=20250303T125023, start_date=20250303T130210, end_date=20250303T130210
[2025-03-03T13:02:10.810+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-03T13:02:10.824+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
